### PR TITLE
[NEW] Separate RegEx Settings for Channels and Usernames validation

### LINKED ISSUE #4301: Separate regEX between usernames & channel name
Right now in the admin settings > general > UTF8, you can change the regEX for an example if i want the users to have atleast 2 letters in their name and max 25 letters. However this apparantly affects the channel names as well.

Could we have two different regEX please? one for usernames and one for channel names.

/Molle

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Proposed changes (including videos or screenshots)

There are now 2 separate settings for validating names - One for **channels** and another for **usernames**.

This change also removes the old `UTF8_Names_Validation` setting and adds 2 new ones `UTF8_User_Names_Validation` and `UTF8_Channel_Names_Validation`.

https://user-images.githubusercontent.com/55396651/111207022-25a23f00-85ef-11eb-8d08-b81f19d27465.mp4

## Issue(s)

Closes #4301

## Steps to test or reproduce

As mentioned in #4301

## Further comments

Please review :slightly_smiling_face:

### CODE REVIEW

**Comment by aditya-mitra** [comment]
@KevLehman Please review :smiley:

File: server/startup/migrations/v219.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Migrations } from '../../../app/migrations';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 219,
```
Comment (KevLehman) [inline server/startup/migrations/v219.js:5]: I'm pretty sure this will clash with new migrations 😬

**Review by KevLehman — CHANGES_REQUESTED** [review]

File: server/startup/migrations/v219.js (hunk 1 above)
Comment (sampaiodiego) [reply server/startup/migrations/v219.js]: Oh my 🙈 sry @aditya-kumawat , I just merged another PR with migration `v223` 😢

Comment (aditya-kumawat) [reply server/startup/migrations/v219.js]: @sampaiodiego You also tagged wrong Aditya here xD

Comment (sampaiodiego) [reply server/startup/migrations/v219.js]: damn github suggestions.. sry the confusion guys 😢

Comment (aditya-mitra) [reply server/startup/migrations/v219.js]: Okay, no problem.
Added migration `v224`

**Comment by aditya-mitra** [comment]
I am sorry. I might have accidentally removed the review from RocketChat/frontend.

**Review by KevLehman — CHANGES_REQUESTED** [review]
Hey! Can you replay your actual changes in the top of a fresh copy of develop? It looks like this PR has a lot of unrelated changes :D

**Pull request closed without merging**